#include <string>

void exitErr(const std::string& cause);